### DISCUSSION TITLE
A few jobs failed

### DISCUSSION THREAD

**Discussion by jayvdb** [lowlighter/metrics#727] [General]
https://github.com/jayvdb/jayvdb/actions/runs/1610880828/attempts/1

Two failing with

```
is at <sha> but expected <sha>
```

Probably not worth worrying about, but I havent seen that before, nor seen errors for quite a while, so thought it worth mentioning.

**Comment by matievisthekat** [comment]
This is probably happening because the version of the repo that the action has is behind the current version. Basically when you try to push changes before pulling. Solution would be to make sure the action is running on the most recent version of your repo.

Those SHAs represent commits in your repo

**Reply by lowlighter** [reply]
Yeah indeed @matievisthekat is right, it occurs when there is a race condition between different jobs pushing at the same time, hence the sha changing out

It's not possible to retry the "push" section yet (the `retries` option is only for rendering) but it would maybe solve this issue.
A workaround for now is to use `needs` and have a linear workflow, though it makes you lost the benefit of parallelization
